bots/pr/src/main/java/org/openjdk/skara/bots/pr/BackportCommand.java

```diff
@@ -409,6 +409,14 @@ public void handle(PullRequestBot bot, HostedCommit commit, LimitedCensusInstanc
                           "$ git commit --message 'Describe additional changes made'\n" +
                           "$ git push " + fork.url() + " " + backportBranchName + "\n" +
                           "```");
+            if (bot.repo().forge().name().equals("GitHub")) {
+                reply.println();
+                reply.print("@");
+                reply.print(realUser.username());
+                reply.print(" ");
+                reply.print("If this is the first time you are using the /backport command for this particular target repository, " +
+                        "a `collaborator` invite will be sent out for my fork [" + fork.name() + "](" + fork.url() + "). You will need to accept this invite before you can proceed.\n");
```

Comment:
If we only print this when the user doesn't have push permission, the message should be something like this instead:

"You are not yet a collaborator in my fork (name). An invite will be sent out and you need to accept it before you can proceed."